wordpress-mobile/WordPress-iOS
# Add Reader screen for unified prologue carousel

Ref #15056. This PR adds the Reader screen to the prologue carousel.

| iPhone | iPad |
|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-29 at 21 53 38](https://user-images.githubusercontent.com/4780/112899587-3f0cb600-90da-11eb-9c47-23fb2b1eabe3.png) | ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-03-29 at 21 55 08](https://user-images.githubusercontent.com/4780/112899596-416f1000-90da-11eb-84c5-5041766f8a79.png) |

*To test*

- Build, run and log out if necessary
- Verify that the Reader (last) page of the login prologue is consistent with the designs (see issue #15056; see also example screenshot above)

PR submission checklist:
